wordpress-mobile/WordPress-iOS
# Display disk quota info for sites

Fixes #8814 

This PR adds the following:
 - Add the the disk quota values to the CoreData blog entity model
 - Updates the quota values by reading them from the WP REST API
 - Display the Quota in the Media Library when tapping on the Add media button
  - Display the Quota in the Site Settings on the new section Media

<img width="390" alt="screen shot 2018-03-05 at 20 56 05" src="https://user-images.githubusercontent.com/651601/37006899-875043bc-20d2-11e8-911f-4735a22c5b8d.png">
<img width="387" alt="screen shot 2018-03-05 at 20 55 56" src="https://user-images.githubusercontent.com/651601/37006900-876d9d9a-20d2-11e8-9d20-2837f5279f0a.png">

Designs where provided by @SylvesterWilmott

To test:
 - Start the app
 - Open the site settings of a WP.com site add see if Disk Quota is displayed
 - Check if on a self-hosted site the quota is not displayed
 - Open the media library for the same site
 - Tap on +
 - Check if the Disk Quota is displayed on the Action Sheet title
 - Check if that info is not displayed on a self-hosted site
